common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -1,7 +1,13 @@
+[CmdletBinding()]
 param(
-    [string]$ProjectFile = './SmokeTest.csproj',
-    [switch]$SkipVersionValidation,
-    [switch]$CI
+    [Parameter(Mandatory=$false)]
+    [string] $ProjectRootPath = ((Resolve-Path "..\").Path),
+
+    [Parameter(Mandatory=$false)]
```

Comment:
Though we're no longer using multiple projects, since we tested these changes, I thought it may be helpful to keep them in the event that we introduce additional projects in the future.   It would save folks the efforts of modding the script.

@danieljurek : Please let me know if you'd prefer to revert to the version in `master`

Reply:
The plan Wes and I have been discussing involves updating versions in projects but likely by directly specified path instead of reading through a top level directory. I think it's best to revert for now (please keep the spelling of "NightlyFeed" which you corrected)